src/envoy/http/service_control/filter_stats.h

```diff
@@ -33,6 +33,8 @@ namespace service_control {
   COUNTER(denied_control_plane_fault)    \
   COUNTER(denied_consumer_blocked)       \
   COUNTER(denied_consumer_error)         \
+  COUNTER(denied_consumer_quota)         \
+  COUNTER(denied_producer_error)         \
```

Comment:
Can we add a doc to describe these statistics?  I don't want people to read code to understand how they are collected.

Reply:
Sure, I'll add to the README.